dissect/target/plugins/apps/shell/powershell.py

```diff
@@ -55,14 +56,15 @@ def powershell_history(self) -> Iterator[ConsoleHostHistoryRecord]:
         for user, _path in self._history:
             file_mtime = _path.stat().st_mtime
 
-            for line in _path.open("r"):
+            for line_number, line in enumerate(_path.open("r")):
                 line = line.strip()
                 if not line:
                     continue
```

Comment:
Should the order not be the actual number of the command, rather than the line number? If we skip a line the order is incremented and it may look like you're missing a command when analyzing the results.